app/helpers/searchHelpers.php

```diff
@@ -432,3 +432,166 @@ function caGetChangeLogForElasticSearch($po_db, $pn_table_num, $pn_row_id) {
 
 		return $va_return;
 	}
+	# ---------------------------------------
+	function caGetQueryBuilderFilters(BaseModel $t_subject, Configuration $vo_query_builder_config) {
+		$vs_table = $t_subject->tableName();
+		$t_search_form = new ca_search_forms();
+		$va_filters = array_values(array_map(
+			function ($pa_bundle) use ($t_subject, $vo_query_builder_config) {
+				return caMapBundleToQueryBuilderFilterDefinition($t_subject, $pa_bundle, $vo_query_builder_config);
+			},
+			$t_search_form->getAvailableBundles($vs_table)
+		));
+		$va_exclude = $vo_query_builder_config->get('query_builder_exclude_' . $vs_table);
+		$va_filters = array_filter($va_filters, function ($vo_filter) use ($va_exclude) {
+			return array_search($vo_filter['id'], $va_exclude) === false;
+		});
+		$va_priority = $vo_query_builder_config->get('query_builder_priority_' . $vs_table);
+		usort($va_filters, function ($po_a, $po_b) use ($va_priority, $vs_table) {
+			$vs_a_id = $po_a['id'];
+			$vs_b_id = $po_b['id'];
+			$vn_a_index = array_search($vs_a_id, $va_priority, true);
+			$vn_b_index = array_search($vs_b_id, $va_priority, true);
+			if ($vn_a_index !== false || $vn_b_index !== false) {
+				// At least one of (a, b) has priority, so the one with highest explicit priority should be first.
+				$vn_a_position = $vn_a_index === false ? 0 : sizeof($va_priority) - $vn_a_index;
+				$vn_b_position = $vn_b_index === false ? 0 : sizeof($va_priority) - $vn_b_index;
+				return $vn_b_position - $vn_a_position;
+			} else {
+				// Neither (a, b) has priority, so look at the tables they reference; there are three cases for each
+				// field specifier:
+				// 1. a field name on its own (no dot), which is implicitly part of the table being searched.
```

Comment:
Yes this is the comment that I referred to above.  Yes this should include access point here under `1`.